### PR TITLE
Feature/i30

### PR DESCRIPTION
**Pull request by TheBlackArroVV**

### CODE REVIEW

File: app/views/organizations/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<% @organizations.each do |organization| %>
+  <ul>
+    <div>
```
Comment (AntonShevchuk) [inline app/views/organizations/index.html.erb]: Remove this `div`

File: app/views/organizations/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+      <li>
+        <div>
```
Comment (AntonShevchuk) [inline app/views/organizations/index.html.erb]: Remove this `div` too

File: app/views/organizations/_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        <% organization.errors.full_messages.each do |message| %>
+          <li><%= message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="container">
+    <%= f.label :name %><br/>
+    <%= f.text_field :name %><br/>
```
Comment (AntonShevchuk) [inline app/views/organizations/_form.html.erb]: You can remove this `<br/>`

**Comment by davinciRor** [comment]
You should use "#No Add...." in commit name. Rebase all please

File: app/controllers/organizations_controller.rb [hunk 4]
```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+class OrganizationsController < ApplicationController
+  before_action :set_organization, only: %i[edit show update destroy]
```
Comment (davinciRor) [inline app/controllers/organizations_controller.rb]: use memorization instead before action

File: app/controllers/organizations_controller.rb [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+  def index
+    @organizations = Organization.all.includes(:user)
```
Comment (davinciRor) [inline app/controllers/organizations_controller.rb]: why do you need includes(:user)?

File: app/controllers/organizations_controller.rb [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+    if @organization.update(organization_params)
+      redirect_to @organization
+    else
+      render :edit
+    end
+  end
+
+  def destroy
+    @organization.destroy
+
```
Comment (davinciRor) [inline app/controllers/organizations_controller.rb]: delete empty line

File: app/views/organizations/_form.html.erb [hunk 7]
```diff
@@ -0,0 +1,27 @@
+<h1>Organization:</h1>
```
Comment (davinciRor) [inline app/views/organizations/_form.html.erb]: Use I18n

File: app/views/organizations/_form.html.erb [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+<%= form_for organization do |f| %>
+  <% if organization.errors.any? %>
+    <div id="error_explanation">
```
Comment (davinciRor) [inline app/views/organizations/_form.html.erb]: Create partial for errors. And add resource like params

File: app/controllers/organizations_controller.rb (hunk 5 above)
Comment (AZatsepa) [reply app/controllers/organizations_controller.rb]: Because he calls ```<%= organization.user.email %>``` in the _index_ page.

Comment (TheBlackArroVV) [reply app/controllers/organizations_controller.rb]: As above and bullet offences

**Pull request merged**